www/js/diary/list.js

```diff
@@ -364,6 +373,10 @@ angular.module('emission.main.diary.list',['ui-leaflet',
     // $scope.expandEarlierOrLater = DiaryHelper.expandEarlierOrLater;
     // $scope.increaseRestElementsTranslate3d = DiaryHelper.increaseRestElementsTranslate3d;
 
+    // rciti start
+    $scope.isAnalyzed = DiaryHelper.isAnalyzed;
+    // rciti end
+
```

Comment:
Ditto. Also, we should store the result of `isAnalysed` as a variable instead of calling a function every time.

Reply:
In fact, there is already an `isDraft` that you can use instead of creating a new function/variable.

```
tripgj.isDraft = $scope.isDraft(tripgj);
```

Reply:
Does the opposite of `isDraft` equal to **analyzed**? If so, i think it is safe to use it.

Reply:
The implementation is not identical, but I think they should have the same effect. Feel free to look at the `isDraft` implementation!